tests/test_open.py

```diff
@@ -116,22 +123,35 @@ def test_memmap(tmp_path):
         assert model.data[6, 19] == new_value
         assert (model.data == new_data).all()
 
-@pytest.mark.parametrize("kwargs", [
-    {"memmap": False}, # explicit False
-    {}, # default
-])
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {"memmap": False},  # explicit False
+        {},  # default
+    ],
+)
 def test_no_memmap(tmp_path, kwargs):
-    data = u.Quantity(np.zeros((400, 400,), dtype=np.float32),
-                      ru.electron/u.s, dtype=np.float32)
-    new_value = u.Quantity(1.0, ru.electron/u.s, dtype=np.float32)
+    data = u.Quantity(
+        np.zeros(
+            (
+                400,
+                400,
+            ),
+            dtype=np.float32,
+        ),
+        ru.electron / u.s,
+        dtype=np.float32,
+    )
```

Comment:
Why is this fully expanded and other groupings are completely compressed? (e.g. test_stdone new lines 76-80)